### DISCUSSION TITLE
Rolling update for KafkaConnect and KafkaMirrorMaker2

### DISCUSSION THREAD

**Discussion by opsocomusr** [strimzi/strimzi-kafka-operator#5521] [Q&A]
When the clients ca certificate is renewed, the cluster operator triggers a rolling update for the kafka brokers and the entity operator. The user certificates are also renewed and updated in the respective secrets. But, the cluster operator does not trigger rolling update for the connect and mirrormaker deployments when the certificate is renewed and it continues to use the older truststore and keystore. Why is it not triggering rolling update for connect and mirrormaker?

**Comment by scholzj** [comment]
I think this is correct. There is probably no hook to roll the deploments when the certs change. Would you mind opening a [Feature Request](https://github.com/strimzi/strimzi-kafka-operator/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=) for it?

**Reply by opsocomusr** [reply]
I have created a feature request https://github.com/strimzi/strimzi-kafka-operator/issues/5524 for this.

**Reply by scholzj** [reply]
Great, thanks.
